File: Códigos/Java/SelectionSort.java
public class SelectionSort {
    public static int[] selectionSort(int[] vetor){
        int tamanho = vetor.length;
        for(int i = 0; i < tamanho; i++){
            int minimo = i;
            for(i +=1; i < tamanho; i++){
                if(vetor[i] < vetor[minimo]){
                    minimo = i;
                }
                vetor[i] = vetor[minimo];
                vetor[minimo] = vetor[i];
            } //fim do for interno
        } //fim do for externo
        return vetor;
    } //fim do método
}

File: Códigos/Python/SelectionSort.py
def selectionSort(vetor):
    tamanho = len(vetor)

    for i in range(tamanho):
        minimo = i
        for i in range(i + 1, tamanho):
            if vetor[i] < vetor[minimo]:
                minimo = i
        (vetor[i], vetor[minimo]) = (vetor[minimo], vetor[i])